docs/tutorial_bracket_strategy.md

```diff
@@ -1,62 +1,62 @@
+If the price moves in way such that both orders are traded one after the other, the passive liquidity provider earns the spread of their orders.
 
-In order to understand this tutorial you should be familiar with the general concept of the programmable-orders.
-Please read the concept described [here](https://docs.google.com/document/d/1_zTkuhWioTbRsbwKnBi5gEi3m7-bFcPyikQid7EUA2M/edit) very carefully.
+This concept is well explained in the documentation of [the programmable-orders](https://docs.google.com/document/d/1_zTkuhWioTbRsbwKnBi5gEi3m7-bFcPyikQid7EUA2M/edit). Please study this article carefully before proceeding with this tutorial.
 
 This tutorial will cover the following steps:
 
 1. Prerequisites
 2. Setting up a MASTER-SAFE
-3. Liquidity deployment script
-4. Withdrawing the tokens
+3. Liquidity provision
+4. Withdrawing liquidity
 5. Even more secure setup with more owners
 
 If you have any questions, don't hesitate to visit us in the [Gnosis discord channel](https://chat.gnosis.io/)
 
 ## Prerequisites
 
-1.  Have a _proposer-account_: You should generate a new Ethereum-key via your preferred tool (e.g. MetaMask) and fund this account with a little bit of Ether. 0.2 ETH should suffice.
-    You will need this account to deploy new proxy-contracts and use it as "proposer account" for the gnosis-safe interface. This will be explained later.
+In this tutorial uses mainly two tools: The normal truffle scripts to build complicated and bundled ethereum transactions and the Gnosis-Safe interface to sign most of these transactions. In the Gnosis-Safe interface, we will generate a MASTER_SAFE, which will own all other accounts required for the liquidity provision and has full control over the funds involved in the liquidity provision.
 
-2.  Have a _signer-account_: You should have one designated signer Ethereum-account. This can be any account, most people would probably prefer to use their usual MetaMask account. This signer account will be added as an owner to your MASTER_SAFE and you will sign transactions with it.
+The prerequisites to this tutorial are:
 
-3.  Investment-tokens: Have the tokens you want to invest into the liquidity provision ready.
+1.  Have a _proposer-account_: You should generate a new Ethereum-account via your preferred tool (e.g., MetaMask) and fund this account with a little bit of Ether. 0.2 ETH should suffice.
+    This account will be used for two tasks: Firstly, it will be used to deploy new proxy-contracts. Secondly, it is used to propose transaction to the Gnosis-Safe interface.
+
+2.  Have a _signer-account_: You should have one designated signer Ethereum-account. This account can be an already given account, and most people would probably prefer to use their usual MetaMask account. This signer-account will be added as an owner to your MASTER_SAFE and you will sign transactions with it.
+
+3.  Investment-tokens: Have the tokens you want to invest in the liquidity provision ready.
```

Comment:
I would say: tokens part of your programmable orders (or whatever it will be called in the end). As it's explicitly NOT investment advice, we should stay clear of the word investment